roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -75,6 +75,15 @@ server {
 
   add_header Strict-Transport-Security "max-age={{ [hsts_max_age, hsts_include_subdomains, hsts_preload] | reject('none') | join('; ') }}";
 
+  {% if item.value.ssl.client_cert_url is defined and item.value.ssl.client_cert_url -%}
```

Comment:
Is this supposed to be `ssl_client_cert_url` in the first one? Just confused why it's checking the same variable

Reply:
`ssl_client_cert_url` was the global option, removed now.

Reply:
With the global option removed, I presume users would no longer disable client verification by setting `client_cert_url: false`. Rather, they simply wouldn't define `client_cert_url` for the relevant site.

If we don't need to check the `bool` interpretation of `client_cert_url`, could we make this change?

```diff
- {% if item.value.ssl.client_cert_url is defined and item.value.ssl.client_cert_url -%}
+ {% if item.value.ssl.client_cert_url is defined -%}
```